app/models/concerns/foreman_bootdisk/orchestration/compute.rb

```diff
@@ -0,0 +1,82 @@
+require 'tmpdir'
+
+module ForemanBootdisk
+  module Orchestration
+    module Compute
+      extend ActiveSupport::Concern
+
+      included do
+        after_validation :queue_bootdisk_compute
+        after_build :rebuild_with_bootdisk
+      end
+
+      def bootdisk_isofile
+        File.join(Dir.tmpdir, "#{name}.iso")
```

Comment:
This could potentially be attacked as tmpdir may be shared and you're `mv`ing over the top with a predictable filename.

It might be best to change this to create a temporary directory ([Dir.mktmpdir](http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html)), store the path in an instance variable (i.e. `@bootdisk_isodir ||= Dir.mktmpdir`) and copy the ISO into that instead.